open-api/rest-catalog-open-api.yaml

```diff
@@ -80,12 +88,14 @@ paths:
         "
         All REST clients should first call this route to get catalog configuration
         properties from the server to configure the catalog and its HTTP client.
-        Configuration from the server consists of two sets of key/value pairs.
+        Configuration from the server consists of:
 
         - defaults -  properties that should be used as default configuration; applied before client configuration
 
         - overrides - properties that should be used to override client configuration; applied after defaults and client configuration
 
+        - capabilities - list of capabilities that are supported by the server
```

Comment:
If there is no `capabilities` payload, then it is assumed to be `["tables"]`?

Reply:
See my other reply [here](https://github.com/apache/iceberg/pull/9940/files#r1523104924).

So I would assume that namespaces/tables are the default behavior without requiring a `tables` capability. Is there any advantage in having `tables` as a capability other than having an option to group the respective endpoints?

Reply:
I'll close this thread and we can discuss on the other one.